src/java/org/apache/cassandra/net/Verb.java

```diff
@@ -259,6 +288,13 @@
     ACCORD_GET_DEPS_RSP         (148, P2, writeTimeout, REQUEST_RESPONSE, () -> GetDepsSerializers.reply, RESPONSE_HANDLER),
     ACCORD_GET_DEPS_REQ         (147, P2, writeTimeout, ACCORD,               () -> GetDepsSerializers.request,       () -> AccordService.instance().verbHandler(), ACCORD_GET_DEPS_RSP),
 
+    CONSENSUS_KEY_MIGRATION_FINISHED(149, P1, writeTimeout, MUTATION, () -> ConsensusKeyMigrationFinished.serializer, () -> ConsensusKeyMigrationState.consensusKeyMigrationFinishedHandler),
```

Comment:
I aligned the first few columns with the ones that are already properly aligned earlier. I couldn't align the second to last parameter because it was already to far to the right.